app/code/Magento/CatalogGraphQl/Model/Resolver/Product/Image.php

```diff
@@ -43,7 +44,7 @@ public function resolve(
         array $args = null
     ): array {
         if (!isset($value['model'])) {
-            throw new \LogicException(__("Cannot resolve entity model"));
+            throw new GraphQlInputException(__('"model" value should be specified'));
```

Comment:
Why? Client knows nothing about the model. It must be passed from the parent resolver. I think before the change we had correct exception, the only change I would made is remove translation:
`throw new \LogicException("Cannot resolve entity model");`

Reply:
https://github.com/magento/graphql-ce/issues/189
Has been created